less/index.less

```diff
@@ -57,3 +57,218 @@ html, body {
     height: 100%;
 }
 
+.flexcontainer{
+	display: flex;
+	justify-content: space-between;
+}
+
+.box{
+	margin: 10px;
+	padding: 3%;
+}
+
+.sectionHeader{
+	font-size: 1.5rem;
+	margin-bottom: 5%;
+}
+
+.single{
+	width: 100%;
+	text-align: center;
+}
+
+.halfpage{
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	width: 45%;
+	height: 50px;
+	margin: 20px;
+	padding: auto;
+	text-align: center;
+	@media (max-width:500px){
+		width: 100%;
+	}
+}
+
+@primary: #ced5e0;
+@secondary: #8e8e8e;
+@third: rgb(245, 245, 245);
+
+body {
```

Comment:
nice work nesting. I would google less examples or you can go look at the repos of past students to see how they are nesting Less.